akka-actor/src/main/resources/reference.conf

```diff
@@ -350,6 +350,19 @@ akka {
         fallback = "fork-join-executor"
       }
 
+      affinity-pool-executor {
+        # The number of threads that this executor will maintain in its pool
+        num-threads = 8
+
+        # The max size of the queue that is used by each thread in the pool
+        # If the queue of a particular worker is full, no more tasks can be
+        # enqueued until space frees up
```

Comment:
And how does the user know what value to specify for this? What is  the tradeoff? Is there a fixed-size buffer or does it use  dynamic growth?